Cargo.toml

```diff
@@ -5,6 +5,7 @@ version = "0.2.5"
 authors = [
         "Flavio Castelli <fcastelli@suse.com>",
         "Rafael Fernández López <rfernandezlopez@suse.com>",
+        "Víctor Cuadrado Juan <vcuadradojuan@suse.com>",
```

Comment:
This could be `Kubewarden Developers <kubewarden@suse.de>`, as once we add the missing, the printed string looks long.

Reply:
No objections, I'm fine replacing that.

Reply:
Done in https://github.com/kubewarden/kwctl/pull/183/commits/90c695d0f3690a0d9f63d2fc7a3252609e00c949.